deckhouse-controller/pkg/helpers/register.go

```diff
@@ -35,6 +36,22 @@ func DefineHelperCommands(kpApp *kingpin.Application) {
 		})
 	}
 
+	{
+		changeRegistryCommand := helpersCommand.Command("change-registry", "Change registry for deckhouse images.")
+		newRegistry := changeRegistryCommand.Arg("new-registry", "Registry that will be used for deckhouse images (example: registry.example.com/deckhouse).").Required().String()
```

Comment:
Could we use actual registry for example?
Also it's not obvious - do we need to pass the scheme or not
```suggestion
		newRegistry := changeRegistryCommand.Arg("new-registry", "Registry that will be used for deckhouse images (example: https://registry.deckhouse.io/deckhouse/ce).").Required().String()
```